specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/virtualNetwork.json

```diff
@@ -1150,11 +1150,11 @@
       "properties": {
         "allowVirtualNetworkAccess": {
           "type": "boolean",
-          "description": "Whether the VMs in the linked virtual network space would be able to access all the VMs in local Virtual network space."
+          "description": "Whether the VMs in the local virtual network space would be able to access the VMs in remote virtual network space."
         },
         "allowForwardedTraffic": {
           "type": "boolean",
-          "description": "Whether the forwarded traffic from the VMs in the remote virtual network will be allowed/disallowed."
+          "description": "Whether the forwarded traffic from the VMs in the local virtual network will be allowed/disallowed in remote virtual network."
```

Comment:
Is this correct? The portal notes that "forwarded traffic" does not originate in the local vnet, so "from the VMs" in the local vnet would appear to be wrong.